handlers/icon_metadata_handler.py

```
@@ -0,0 +1,29 @@
+from usecases.metadata import Metadata
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def handle_get(
+    event: ApiGatewayEvent, _context: LambdaContext, metadata: Optional[Metadata] = None
+) -> dict:
+    metadata = metadata or Metadata()
+    if "id" not in event.query:
+        raise ApiError("invalid_parameters")
+    hash = event.query["id"]
```

Ruff (0.15.10)

[error] Variable `hash` is shadowing a Python builtin

(A001)